### PR TITLE
Add details about `AllowedCalls`

### PR DESCRIPTION
**Pull request by b00ste**
#  What does this PR introduce?

- Add details about `AllowedCalls`

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-8630 Improve documentation around `AllowedCalls`](https://app.clickup.com/t/2645698/DEV-8630)

**Comment by Hugoo** [comment]
Task linked: [DEV-8630 Improve documentation around `AllowedCalls`](https://app.clickup.com/t/2645698/DEV-8630)

**Comment by Hugoo** [comment]
Task linked: [DEV-8630 Improve documentation around `AllowedCalls`](https://app.clickup.com/t/2645698/DEV-8630)

**Comment by Hugoo** [comment]
Task linked: [DEV-8630 Improve documentation around `AllowedCalls`](https://app.clickup.com/t/2645698/DEV-8630)

File: docs/contracts/overview/KeyManager.md [hunk 1]
```diff
@@ -21,6 +21,28 @@ To illustrate, in the case of Universal Profile and the browser extension, you c
...
+Supposedly we have the following `AllowedCalls`:
+`0x002000000002cafecafecafecafecafecafecafecafecafecafe24871b3d7f23690c002000000003cafecafecafecafecafecafecafecafecafecafe24871b3d44c028fe`
+
+It can be decoded as:
+
+| Allowed Calls | |
+|:-----|:-----|
+| First allowed call | Call Types - `0x00000002` (Call) <br/> Address - `0xcafecafecafecafecafecafecafecafecafecafe` <br/> Standard - `0x24871b3d` (LSP0) <br/> Function - `0x7f23690c` (`setData(bytes32,bytes)`) |
+| Second allowed call | Call Types - `0x00000003` (Transfervalue & Call) <br/> Address - `0xcafecafecafecafecafecafecafecafecafecafe` <br/> Standard - `0x24871b3d` (LSP0) <br/> Function - `0x44c028fe` (`execute(uint256,address,uint256,bytes)`) |
+
```
Comment (YamenMerhi) [inline docs/contracts/overview/KeyManager.md:45]: Maybe you can add here a sentence after the table to explain that in these cases the user is allowed to call and transfer the address cafecafe if it supports the standard `0x24871b3d` using ERC165 and can only interact with the execute function ..
Something like this

**Review by YamenMerhi — APPROVED** [review]

**Review by CJ42 — APPROVED** [review]

**Pull request merged**